core/src/main/java/nl/nn/adapterframework/jdbc/migration/Migrator.java

```diff
@@ -58,10 +58,13 @@ public void configure(Configuration configuration) throws ConfigurationException
 	}
 
 	public synchronized void configure(Configuration configuration, String changeLogFile) throws ConfigurationException {
+		AppConstants appConstants = AppConstants.getInstance(configuration.getClassLoader());
 		setName("JdbcMigrator for configuration["+ configuration.getName() +"]");
+		if(StringUtils.isEmpty(getDatasourceName())) {	
```

Comment:
hij is toch altijd leeg? Denk dat deze if weg kan.

Reply:
Niet als je hem met migrator.setDatasourceName() gezet hebt.